lib/apiservers/engine/backends/container_test.go

```diff
@@ -615,7 +616,7 @@ func TestPortInformation(t *testing.T) {
 	mockContainerInfo := &plmodels.ContainerInfo{}
 	mockContainerConfig := &plmodels.ContainerConfig{}
 	containerID := "foo"
-	mockContainerConfig.ContainerID = &containerID
+	mockContainerConfig.ContainerID = containerID
```

Comment:
This is the least problematic change you can think of - at least for these values I was able to check the code surrounding it. The most scariest part is when I set "x-nullable":false - this is where zero values are no longer accepted. I tried to be careful with them, but things that are changed are not as scary as those that are unchanged.